_events/2017-08-05.md

```diff
@@ -20,7 +20,7 @@ Stay tuned to our blog, twitter, or facebook for more announcements!
 - 1:00 PM - Hacking, Brainstorming session, and Workshops start
 - 8:00 PM - Concert / Demos
 
-# Talks
+# Talks 
```

Comment:
Please revert this file:

$ git checkout origin/master [filename]

Reply:
Good to know, thanks!